ts/features/walletV3/payment/screens/WalletPaymentMakeScreen.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
```

Comment:
I think a name like `WalletPaymentScreensPager` could make the flow easier to understand on first sight, since this technically is more of a "screen picker" than a screen by itself.
What do you think?

Reply:
Technically it is a screen with a `ViewPager` which changes the component rendered inside the screen. Logically is a screen where the user makes the payment. This is the reasoning behind the name.